documentation/index.md

```diff
@@ -6,12 +6,13 @@ title: Documentation
 <div class="row">
   <div class="col-md-9">
     <div class="well">
-      <h4><a href="usecases">Use Cases</a></h4>
+      <h4><a href="/usecases">Use Cases</a></h4>
```

Comment:
In FF, if I try to go to http://stixproject.github.io/documentation, it redirects me to http://stixproject.github.io/documentation/ (add the trailing slash), so the relative link "usecases" goes to http://stixproject.github.io/documentation/usecases, which doesn't exist. The "Go" button already worked; this was for the heading above it.